lib/CryptoLD.js

```diff
@@ -67,10 +67,48 @@ class CryptoLD {
     return this.suites.get(type).from(serialized);
   }
 
+  async fromKeyId({
+    keyId, documentLoader, checkContext = true, checkRevoked = true
+  } = {}) {
+    if(!keyId) {
+      throw new TypeError('Key ID is required.');
```

Comment:
Normally something like `"The "keyId" parameter is required.`, same for `documentLoader`

Reply:
Thanks, fixing.